api/app/hfi/fire_centre_cache.py

```diff
@@ -0,0 +1,47 @@
+""" Redis cache wrapper for hydrates fire centres
+    We can safely cache the fire centres, as they don't change them very often.
+    the eco-division logic is very slow, and chomps up 2 seconds!
+"""
+
+import json
+import logging
+from typing import Optional
+from app.schemas.hfi_calc import HFIWeatherStationsResponse
+from app.utils.redis import create_redis
+
+logger = logging.getLogger(__name__)
+cache_expiry_seconds = 86400
```

Comment:
Nice to have: a comment quantifying 86400 seconds in a more human-meaningful value (e.g., x days)